client.sh

```diff
@@ -64,12 +64,14 @@ run() {
     rm -f ${pipe_name}
 
     # Print the status and collect stats
-    echo "${run_id}: ${status}!"
-    sleep 15
-    ${git_dir}/stats.sh \
+    echo "Run completed for ${run_id}. Waiting and collecting stats in 1 min"
+    sleep 45
```

Comment:
Why do we need to wait for 1 minute?

Reply:
The process's stats file are not immediately available after we echo stop to process. They will need to write the result into pkl files. 1 minute is done to be extremely conservative.